groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -3910,20 +3609,29 @@ class optional<TYPE, false> {
 // }}} END GENERATED CODE
 #endif
 
-    //MANIPULATORS
+    // MANIPULATORS
 #if !BSLS_COMPILERFEATURES_SIMULATE_CPP11_FEATURES
     template <class... ARGS>
     void emplace(BSLS_COMPILERFEATURES_FORWARD_REF(ARGS)...);
-        // Destroy the current 'value_type' object, if any, and create a new
-        // one using the provided arguments.
+        // Assign to this 'optional' object the value of the (template
+        // parameter) 'TYPE' created in place using the specified 'il' and
+        // specified 'args'.  If this 'optional' object already contains an
+        // object ('false == isNull()'), that object is  destroyed before the
+        // new object is created.  Note that if the constructor of 'TYPE'
+        // throws an exception this object is left in a disengaged state.
 
 #if defined(BSLS_COMPILERFEATURES_SUPPORT_GENERALIZED_INITIALIZERS)
     template <class INIT_LIST_TYPE, class... ARGS>
     void emplace(std::initializer_list<INIT_LIST_TYPE>,
```

Comment:
```suggestion
    void emplace(std::initializer_list<INIT_LIST_TYPE> il,
```

Please name the `initializer_list` argument in declarations so that contracts mentioning `il` make sense (here and in other similar cases).

Reply:
done